src/pages/library/dataBrowser-utils.js

```diff
@@ -65,7 +98,8 @@ const extractTags = snapshot => {
       snapshot.samples?.disease,
       snapshot.dataType,
       snapshot.dataModality,
-      _.map('dcat:mediaType', snapshot.files)
+      _.map('dcat:mediaType', snapshot.files),
+      snapshot.dataReleasePolicy.tag
```

Comment:
With the above change this would be:
```suggestion
      snapshot.dataReleasePolicy.policy
```